src/bin/git-secure-fetch.rs

```diff
@@ -32,8 +32,8 @@ fn main() {
         Some(v) => v.to_owned(),
     };
 
-    let branch = match matches.value_of("BRANCH") {
-        None => panic!("Must supply a BRANCH argument"),
+    let reference = match matches.value_of("REFERENCE") {
+        None => panic!("Must supply a REFERENCE argument"),
```

Comment:
It would be nice to avoid panics. You could use `bail!` from error chain if you moved the logic in main to another function and then called `handle_error` from main?

Reply:
I think that this specific case should probably go into the issue tracker and be addressed as part of a different PR, since this one is really just supposed to be adding tag support.